### ISSUE TITLE
Cross-platform support

### ISSUE THREAD

**Issue by hameerabbasi** [conda-forge/gdb-feedstock#6]
I find that I need this a lot to debug (for example) Cython. Is this within scope?

**Comment by gqmelo** [comment]
Hi @hameerabbasi . I never debugged Cython code, but I guess you could use gdb for that.

When I was adding `gdb` to conda-forge I tried to build for MacOS too, but faced some problems. As I don't have a Mac and there was nobody else interested, so we just published the Linux build. I know `gdb` is also buildable on `Windows` but not sure how useful would be.

If you would like to submit a PR I'm glad to help with any issues you find, but I'm afraid I cannot do much more than that.

**Comment by jakirkham** [comment]
Yeah there are lots of options for using `gdb` with Cython. Maybe you have already seen this, but [this doc page from Cython]( http://cython.readthedocs.io/en/latest/src/userguide/debugging.html ) is pretty helpful.

This would be better still if we had debug builds of Python. That is an open issue ( https://github.com/conda-forge/python-feedstock/issues/73 ), but may be easier to solve with `conda-build` 3.

Opened issue ( https://github.com/conda-forge/gdb-feedstock/issues/7 ) on macOS builds. If you have any details from your last attempt @gqmelo, can look into doing this.

**Comment by bilderbuchi** [comment]
Would offering a Windows build of `gdb` be easily possible? 

I have just managed to successfully assembled the toolchain for building a mixed C++/Fortran cmake project using only conda-forge! This needs basically only to `m2w64-toolchain cmake doxygen` 🍾 I like the conda environment for offering these outside-Python tools.
Unfortunately, I just realised that I lack a debugger, so that kinda put a damper on my ambitions. Surprisingly, the m2w64 tools include a load of compilers, but no `gdb`, as opposed to installing vanilla mingw from their website. :-(

**Comment by jakirkham** [comment]
Would you be willing to make a PR?

**Comment by bilderbuchi** [comment]
I can certainly try. 
However, a cursory examination shows that you seem to need [libcxx pinned to v10](64201c95e1e120be8e7b07fa38c509b3c1fedd53) for building, but its Windows version seems [stuck on v7.0](https://anaconda.org/conda-forge/libcxx) -- I don't know how much I can do about this. (Apparently you yourself [were involved](https://github.com/conda-forge/libcxx-feedstock/issues/8) in getting the Windows builds off the ground in the first place)

**Comment by jakirkham** [comment]
The first link doesn't work for me. In any event we typically use `libcxx` on macOS (not Windows). So wouldn't worry about that

**Comment by bilderbuchi** [comment]
ah, GH mangled that in a wrong way, link fixed. 
OK, so I'll investigate how to exclude this build required based on platform, let's see how far I get...
